tests/Unit/Utilities/Test_Functional.cpp

```diff
@@ -468,6 +475,19 @@ SPECTRE_TEST_CASE("Unit.Utilities.Functional", "[Utilities][Unit]") {
   ERROR("Failed to trigger ASSERT in an assertion test");
 #endif
 }
+
+    // clang-format off
```

Comment:
I didn't realize this works! good to know :) I never understood why clang-format gets confused by our error tests...